.circleci/config.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           command: yarn prettier
       - run:
           name: Unit Tests
-          command: yarn test --maxWorkers=2
+          command: yarn test
```

Comment:
I think you're hitting an issue with running unit tests before these packages are built (so, the @fluentui/digest/lib/index.js has been generated yet). Either we'll have to swap things around here or teach jest about how to resolve the modules (other packages use lerna-alias for this purpose I think?)